5_databases/week1/lesson1(HW).sql

```diff
@@ -0,0 +1,43 @@
+-- Active: 1699540080106@@0.0.0.0@3306@hyf_lesson1
+
+SELECT COUNT(*)
+FROM task;
+
+SELECT COUNT(*)
+FROM task
+WHERE due_date IS NULL;
+
+select task.title
+FROM task
+WHERE status_id = 3;
+
+SELECT task.title
+FROM task
+WHERE status_id != 3;
+
+SELECT *
+FROM task
+ORDER BY created DESC;
+
+SELECT *
+FROM task
+ORDER BY created DESC
+LIMIT 1;
+
+SELECT task.title, task.due_date
+FROM task
+WHERE title LIKE "%database%";
```

Comment:
Well done but you also have to check the description field for database. Do this by using the `OR` operator